cpe/openscap-cpe-oval.xml

```diff
@@ -699,6 +748,20 @@
                 <pattern operation="pattern match">^VERSION=.([[:digit:]]*)</pattern>
                 <instance operation="greater than or equal" datatype="int">1</instance>
             </textfilecontent54_object>
+            <file_object xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix" version="1" id="oval:org.open-scap.cpe.amznlinux-release:obj:1" >
+                <filepath>/etc/os-release</filepath>
+            </file_object>
+            <textfilecontent54_object
```

Comment:
WRLinux parses /etc/os-release because we do not require rpm/yum/dnf to be present.  Other package managers may used instead, or in some cases, no PM at all.  That's not really appropriate for something based off RHEL or CentOS, or possibly for any non-embedded deployment.